### DISCUSSION TITLE
Is it possible to obsfucate the password in authWithPassword?

### DISCUSSION THREAD

**Discussion by JunoNgx** [pocketbase/pocketbase#4685] [Q&A] [answered]
Upon calling `client.authWithPassword("EMAIL_OR_USERNAME", "PASSWORD")` to [authenticate](https://pocketbase.io/docs/authentication), I notice that the `auth-with-password` `POST` request sends the password in plain text in the request's JSON body.

```
{
	"identity": "john@gmail.com",
	"password": "12341234"
}
```

Is it possible to do something about this from my side to make it less of a security issue?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure I follow. We need the plain password so that we can verify its hash.
What do you mean by _"less of a security issue"_? If you use HTTPS the submitted data would be encrypted anyway if that is your concern.

**Reply by JunoNgx** [reply]
Have been reading up more on this. Yes, looks like this is the standard practice and it's probably safe to do so. Thank you for the confirmation and answering.
